### DISCUSSION TITLE
First tutorial step failing

### DISCUSSION THREAD

**Discussion by jon-n** [vercel/next.js#17845] [Help]
Feels a little silly to have this problem, but I've run into an issue with the first step of the tutorial.  When I do the command to create a new app with the starter code,

`npx create-next-app nextjs-blog --use-npm --example "https://github.com/vercel/next-learn-starter/tree/master/learn-starter"`

I get the following error:

`Could not locate the repository for "https://github.com/vercel/next-learn-starter/tree/master/learn-starter". Please check that the repository exists and try again.`

I saw a similar discussion here where someone else was running into a similar error, but there didn't seem to be a resolution there either.

For what it's worth, I'm on node 14.13.1, npm 6.14.8, and on a Mac in zsh.

**Comment by DjRAST** [comment]
+1

**Comment by 25x8** [comment]
Still not working. Best support ever.

**Reply by lichenbuliren** [reply]
+1, Still not working. Best support ever.

**Comment by michrome** [comment]
Try following this [guide to running Next.js](https://www.j4e.name/articles/running-nextjs-on-a-new-m1-mac), which I’ve tested recently.

(It says M1 but is just as applicable to Intel too).

**Comment by kashaf-ahmed-drishyaai** [comment]
Make sure you have Node.js version 10.13 or later. you can use nvm to switch between different node versions
